contracts/cw721-base/src/upgrades/v0_17.rs

```diff
@@ -1,27 +0,0 @@
-use cosmwasm_std::{CustomMsg, DepsMut, Response};
-use cw721_base_016 as v16;
-use serde::{de::DeserializeOwned, Serialize};
-
-use crate::ContractError;
-
-pub fn migrate<T, C, E, Q>(deps: DepsMut) -> Result<Response<C>, ContractError>
-where
-    T: Serialize + DeserializeOwned + Clone,
-    Q: CustomMsg,
-    E: CustomMsg,
-{
-    // remove old minter info
-    let tract16 = v16::Cw721Contract::<T, C, E, Q>::default();
-    let minter = tract16.minter.load(deps.storage)?;
-    tract16.minter.remove(deps.storage);
-
-    // save new ownership info
-    let ownership = cw_ownable::initialize_owner(deps.storage, deps.api, Some(minter.as_str()))?;
```

Comment:
Also this old migrato logic of minter to ownership is error prone. Like it is e.g. possible to migrate from v0.16 to v0.18. Then minter is transferred to new addr. Keep in mind in old minter store there is still old addr. So migrating back to v0.16 and to v0.18 again, would reset minter addy. Better to always check whether new minter ownship is set - if so no migration needed. 

This is an edge case, but still not good.

Reply:
We needed this for migration to v.01.7. I think I might have been the one that "wrote" it but I was just copying migration logic from cw base contract.